_posts/2024-04-18-SceneMaps Bug Fix.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+# If your post title is longer or more complicated
+# than can be represented in the filename, uncomment the following line
+# and specify a custom title
+# title:  "SceneMaps Bug Fix"
+
+# Uncomment only one of the below categories
+categories: 
+ Bug Fix
+#- Contribution
+
+
+# Enter your name below
+author: Erin Ford
+---
+
+For my bug fix, I decided to take a couple friends' of mine's project, give it a proper open source license, and help fix a few of the listed connections.
+
+## What is SceneMaps
+SceneMaps is a project to map out the various connections between local underground music scenes, and tie them together with various shared band members and shows they have played together.
+
+
+## Why I picked it
+I'm very much involved in the local rochester music scene, so I found the concept to be interesting.
+
+## Avaliable Resources
+- Them explaining the general gist to me
+- basically nothing else
+
+## The Issue:
+The repo's list of bands was not up to date, and the entire repo did not have any licenses. That was mostly what I could do the fastest while I read through the code and got an idea of what future way I could contribute.
+It was relatively simple, I just updated whatever pieces of information I already knew, searched through the dependencies and tech that it used to see if there were any conflicting licenses with the one we agreed on (which was the Hippocratic License), and then pushed the license to the repo.
+As for updating things, They walked me through how to do it nd I pushed those changes on their computer. I got these accepted easily, it's only a team of 3 people so I was able to just easily get commit access.
+
+Commit Links:
+[Licensing](https://github.com/Scene-Links/Scene-Links/commit/ad8579790b3bc2bdccd3b0ca3452cd433faff130)
+[Name corrections](https://github.com/Scene-Links/Scene-Links/commit/5f5aec843ec184d097215cc693967d9a98fc1e6b)
+[Name additions](https://github.com/Scene-Links/Scene-Links/commit/99b3227cea216cb0bc86d24ddc618cb28d488b89)
```

Comment:
Seems like markdown wants to put this all on one line.

maybe try either adding them to a bullet list (see below):

 ```suggestion
Commit Links:

- [Licensing](https://github.com/Scene-Links/Scene-Links/commit/ad8579790b3bc2bdccd3b0ca3452cd433faff130)
- [Name corrections](https://github.com/Scene-Links/Scene-Links/commit/5f5aec843ec184d097215cc693967d9a98fc1e6b)
- [Name additions](https://github.com/Scene-Links/Scene-Links/commit/99b3227cea216cb0bc86d24ddc618cb28d488b89)
```

or maybe adding spacing:
```suggestion
Commit Links:

[Licensing](https://github.com/Scene-Links/Scene-Links/commit/ad8579790b3bc2bdccd3b0ca3452cd433faff130)

[Name corrections](https://github.com/Scene-Links/Scene-Links/commit/5f5aec843ec184d097215cc693967d9a98fc1e6b)

[Name additions](https://github.com/Scene-Links/Scene-Links/commit/99b3227cea216cb0bc86d24ddc618cb28d488b89)
```

Those should show up as quick suggestions that you can 1-click commit in case that makes things a little easier.